src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -698,16 +707,27 @@ export class MgtPeoplePicker extends MgtTemplatedComponent {
             }
           }
         }
-
+        // default group search with user input
```

Comment:
This logic should be combined with the similar logic for default people above

Reply:
refactored here: 

2f7b3d6

Reply:
make sure to remove comment